tests/test_views.py

```diff
@@ -0,0 +1,136 @@
+import pytest
```

Comment:
Do we really want to go fully functional with the tests? I honestly prefer the way we can namespace the tests with classes.

Reply:
@lucifurtun if you feel like test classes are best suited for your needs, I personally don't see any problem in using them.

Reply:
ok